mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/gridfs/GridFSTest.java

```diff
@@ -153,6 +164,8 @@ public void canRemoveEntityWithBlob() {
 			assertBlobAreEqual( photo.getContentAsBlob(), BLOB_CONTENT_1 );
 			assertThat( photo.getContentAsByteArray() ).isNotNull();
 			assertThat( photo.getContentAsByteArray() ).isEqualTo( BLOB_CONTENT_1 );
+			assertThat( photo.getContentAsString() ).isNotNull();
```

Comment:
This line is not needed, the  isEqualTo already take care of null values

Reply:
Ok. I will remove excess line.